app/webpacker/css/admin_v3/components/tom_select.scss

```diff
@@ -43,15 +37,23 @@
   }
 
   .ts-dropdown {
+    // Unstyled, we style the dropdown-content instead
+    margin: 0;
+    background: none;
+    border: none;
+    box-shadow: none;
+  }
+
+  .ts-dropdown-content {
     margin-top: 4px;
     color: $near-black;
+    background-color: $color-body-bg;
     @include border-radius($border-radius);
-    border: none;
     box-shadow: $shadow-dropdown;
 
-    .ts-dropdown-content {
-      padding: $border-radius 0;
-    }
+    // .ts-dropdown-content {
+    //   padding: $border-radius 0;
+    // }
```

Comment:
Should be deleted instead of commented out ?

Reply:
Oops, yes I amended the commit but apparently never pushed it up! 
Pushed up now, plus another small commit.